src/python/grapl-model-plugin-deployer/src/grapl_model_plugin_deployer.py

```diff
@@ -75,6 +75,10 @@
     )["SecretString"]
 
 app = Chalice(app_name="model-plugin-deployer")
+try:
+    app.debug = bool(os.environ.get("CHALICE_DEBUG", False))
+except Exception as e:
+    LOGGER.warn(e)
```

Comment:
Let's remove this block for now, I'm going to add debugging support for Chalice in a separate PR.

Reply:
removed